### PR TITLE
Adapt 8235 issue with first flipcard being focused when there are multiple flipcards

### PR DESCRIPTION
**Pull request by eleanor-heath**

### CODE REVIEW

**Review by ethan-lp — APPROVED** [review]
👀

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by ethan-lp — APPROVED** [review]
👀

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by ethan-lp — APPROVED** [review]
👀

**Review by ethan-lp — APPROVED** [review]
👀

File: js/adapt-contrib-flipcard.js [hunk 1]
```diff
@@ -1,191 +1,238 @@
...
+  preRender() {
+    this.model.set('_items',
+      this.model.get('_items').map(obj => ({
+        ...obj,
+        backBodyText: $(obj.backBody).text()
+      }))
+    );
+    // is apple platform used to determine if voiceover is being used
+    const platform = navigator?.platform?.toLowerCase();
+    this.isApplePlatform = (platform === 'mac' || platform === 'macintel' || platform === 'iphone' || platform === 'ipad');
```
Comment (cahirodoherty-learningpool) [inline js/adapt-contrib-flipcard.js]: I think you can use
`if (Adapt.device.isAppleDevice() || Adapt.device.getOperatingSystem() === 'mac')`
It uses the same logic as what you are writing, but if it gets improved in the future we will be falling back to it

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Pull request merged**